src/utils.rs

```diff
@@ -30,7 +30,7 @@ impl Default for OkModule {
 
 impl OkModule {
     //assumes only one function
-    pub fn build(wasm_bin: Vec<u8>, infos: &impl LineInfos) -> Result<Self> {
+    pub fn build(wasm_bin: Vec<u8>, infos: &impl LineInfos, synthetic_ends: usize) -> Result<Self> {
```

Comment:
I think this synthetic_ends should be exposed by the LineInfos trait so the info can be bound together (rather than making an extra argument here).